src/stdatamodels/jwst/datamodels/schemas/conv_kernel.schema.yaml

```
@@ -0,0 +1,22 @@
+%YAML 1.1
+---
+$schema: "http://stsci.edu/schemas/asdf/asdf-schema-1.0.0"
+id: "http://stsci.edu/schemas/jwst_datamodel/conv_kernel.schema"
+allOf:
+- $ref: referencefile.schema
+- type: object
+  properties:
+    data:
+      title: NIR Optimized Convolution Kernel Fourier Coefficients
+      type: object
+      properties:
+        meta:
+            type: object
+        frequencies:
+            type: object
+            description: |
+               Detector name, gamma, and zeta values
+        subarray_cases:
+            type: object
+            description: |
+               Fourier Coefficients for NIR Optimized Convolution Kernel
```

```suggestion
  patternProperties:
    "nrc[ab][1-4]":
      type: object
      properties:
        gamma:
          datatype: float64
        zeta:
          datatype: float64
      required: [gamma, zeta]
```
This suggestion matches the reference file shared in https://jira.stsci.edu/browse/CRDS-892
This suggestion is also very nircam specific. If it's worth pursuing it could be extended to other detectors. We will want to standardize the detector naming.